src/hook_files/hook_script.sh

```diff
@@ -21,5 +21,5 @@ else
   ensureMinimumRustyHookCliVersion || true
 fi
 
-rusty-hook run --hook "${hookName}" "${gitParams}"
+rusty-hook run --hook "${hookName}" $([ -z "$gitParams" ] && echo "-- $gitParams" || echo "")
```

Comment:
ShellCheck is barking but we definitely don't won't to add quotes as it suggests, an ignore is definitely appropriate in this case

```suggestion
# shellcheck disable=SC2046
rusty-hook run --hook "${hookName}" $([ -z "$gitParams" ] && echo "-- $gitParams" || echo "")
```

Reply:
Was already on it :+1:

Reply:
I uh... forgot to push the change.